### PR TITLE
Update swagger dependency. Fix roslyn code smell.

### PR DESCRIPTION
**Pull request by Desomph**
Updated the swagger to the newest version (v12.0.3 -> v12.3.1).
Changed couple of lines based on roslyn analyzer.

### CODE REVIEW

File: Exebite.API/Startup.cs [hunk 1]
```diff
@@ -112,11 +108,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseMvc();
 
-            // todo update the swagger to new definiton
-            app.UseSwaggerUi(typeof(Startup).GetTypeInfo().Assembly, settings =>
-            {
-                settings.GeneratorSettings.DefaultPropertyNameHandling = PropertyNameHandling.CamelCase;
```
Comment (stanoje) [inline Exebite.API/Startup.cs]: Did you checked the generated swagger file?

**Comment by stanoje** [comment]
Have you checked the generated swagger file? Is the casing the same as it was?

**Comment by Desomph** [comment]
Looks the same to me, URL is the same `swagger/v1/swagger.json`
Example of the the generated API, it's still camel case (guess they default to it):

```JSON
"/api/CustomerAliases/{id}": {
      "put": {
        "tags": [
          "CustomerAliases"
        ],
        "operationId": "CustomerAliases_Put",
        "consumes": [
          "application/json-patch+json",
          "application/json",
          "text/json",
          "application/*+json"
        ],
        "parameters": [
          {
            "type": "integer",
            "name": "id",
            "in": "path",
            "required": true,
            "format": "int32",
            "x-nullable": false
          },
          {
            "name": "model",
            "in": "body",
            "required": true,
            "schema": {
              "$ref": "#/definitions/UpdateCustomerAliasDto"
            },
            "x-nullable": false
          }
        ],
        "responses": {
          "200": {
            "x-nullable": true,
            "description": "",
            "schema": {
              "type": "file"
            }
          }
        }
      },
      "delete": {
        "tags": [
          "CustomerAliases"
        ],
        "operationId": "CustomerAliases_Delete",
        "parameters": [
          {
            "type": "integer",
            "name": "id",
            "in": "path",
            "required": true,
            "format": "int32",
            "x-nullable": false
          }
        ],
        "responses": {
          "200": {
            "x-nullable": true,
            "description": "",
            "schema": {
              "type": "file"
            }
          }
        }
      }
    },
```

**Comment by Desomph** [comment]
Not ready yet.

**Pull request closed without merging**